packages/open-lens/webpack/renderer.ts

```diff
@@ -113,6 +114,15 @@
       filename: "[name].css",
     }),
 
+    new CopyPlugin({
+      patterns: [
+        {
+          from: "node_modules/@k8slens/core/static/build/library/*.ttf",
+          to: "[name][ext]",
```

Comment:
_nit:_
```suggestion
          to: "fonts/[name][ext]",
```

Reply:
I would love to put them under `fonts/` but it means that `core` package also should have fonts under `fonts/`. I will leave this to follow up PRs.